acceptance/config/nodes/lucidx64.yaml

```diff
@@ -3,15 +3,15 @@ HOSTS:
     roles:
```

Comment:
I think that's in-line with Kylo's suggestion to use puppet-agent platform names; however, it's slightly different.

Puppet-Agent uses `ubuntu-10.04-amd64`, vmpooler uses ubuntu-1004-x86_64. So some naming sanity would be nice here; I'm tempted to follow the vmpooler format, because it's likely harder to change than the puppet-agent configuration.